src/main/webapp/app/pages/curation/header/MutationsSectionHeader.tsx

```diff
@@ -46,6 +46,7 @@ export enum SortOptions {
   LAST_MODIFIED = 'Last Modified',
   POSITION_INCREASING = 'Position Increasing',
   POSITION_DECREASING = 'Position Decreasing',
+  FIREBASE = 'Firebase',
```

Comment:
I assume this will make the sort method returning 0 and it will use the order in the list?

Reply:
Yeah exactly